data_structures/is_multiple.py

```diff
@@ -0,0 +1,21 @@
+# is multiple of a number.
+# numbers n and m, where n = mi.
+# return true if n is a multiple of m else return false.
+
+
+def is_multiple(n, m):
+    ret_val = False
+    for i in range(1, n + 1):
```

Comment:
Don't need a loop to do this.

Reply:
The for loop is removed and mod operator is used to check if the numbers are multiples of each other.